src/index.js

```diff
@@ -0,0 +1,7 @@
+// import _ from 'lodash';
+import './style.css';
+import './modules/function.js';
+// import { renderTask, renderForm } from './modules/data.js';
+
+// renderTask();
+// renderForm();
```

Comment:
- [ ] Leaving unused code as a comment makes our codebase unclean and a bit cluttered. Kindly remove the commented pieces of code to make the repo more professional. 🤓️